.circleci/config.yml

```diff
@@ -268,41 +267,212 @@ jobs:
           path: /tmp/license-scanner/results
           prefix: licenses
 
-  build:
+  build-inbound-local:
     executor: default-machine
     steps:
       - checkout
       - run:
-          name: Build Docker $CIRCLE_TAG image
+          name: Setup Docker env variables
+          command: |
+            echo 'export DOCKER_ORG=`cat package.json | jq -r .name | sed s/@// | sed s/$/-inbound/ | sed s/\\\\/.*//`' >> $BASH_ENV
+            echo 'export CIRCLE_PROJECT_REPONAME=`cat package.json | jq -r .name | sed s/@// | sed s/$/-inbound/ | sed s/\.*\\\\///`' >> $BASH_ENV
+      - run:
+          name: Build Docker inbound:local
           command: |
-            echo "Building Docker image: $CIRCLE_TAG"
-            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$CIRCLE_TAG .
+            echo "Building Docker image-inbound: local"
+            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME/inbound:local -f ./docker/Dockerfile.Inbound ./
```

Comment:
Can you have another slash like this? 

e.g. `mojaloop/thirdparty-scheme-adapter/inbound:local` doesn't look like a proper docker image name + tag

Reply:
@lewisdaly yes there is more - let me fix it